src/test/java/org/sundry/recipes/model/CategoryTest.java

```diff
@@ -27,9 +30,21 @@ public void getId() {
   
   @Test
   public void getCategoryName() {
+    String categoryName = "testCat";
+    category.setCategoryName(categoryName);
+    assertEquals(categoryName, category.getCategoryName());
```

Comment:
Try using hamcrest. It would read something like:
```assertThat(category.getCategoryName, equalTo(categoryName)```

Reply:
I tried this in some tests and used hamcrest in rest of the tests.